transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannel.java

```diff
@@ -33,18 +35,18 @@
     private volatile DomainSocketAddress remote;
 
     public EpollDomainSocketChannel() {
-        super(Native.socketDomainFd());
+        super(newSocketDomain());
     }
 
-    public EpollDomainSocketChannel(Channel parent, FileDescriptor fd) {
+    public EpollDomainSocketChannel(Channel parent, Socket fd) {
```

Comment:
one of these days I'm going to get you to say "ok...fine...we can break API" :wink: